### ISSUE TITLE
Redundant bias paramter in ResNet's conv2d.

### ISSUE THREAD

**Issue by sebastian-sz** [keras-team/keras-cv#629]
**Short description**
In `tf.keras` by default Conv2D has bias enabled. We should explicitly set `use_bias=False` if the convolution layer is followed by BatchNormalization.

This is the done in some existing Keras-CV implementations:
* DarkNet Conv Block - [link](https://github.com/keras-team/keras-cv/blob/f3e32b5d74d69419c5ec17b04f9ca96807474ed1/keras_cv/models/__internal__/darknet_utils.py#L28)
* DenseNet - [link](https://github.com/keras-team/keras-cv/blob/f3e32b5d74d69419c5ec17b04f9ca96807474ed1/keras_cv/models/densenet.py#L134)
* ResNet V2 - [link](https://github.com/keras-team/keras-cv/blob/f3e32b5d74d69419c5ec17b04f9ca96807474ed1/keras_cv/models/resnet_v2.py#L116)

But not in `resnet_v1` - all ResNet's have [bias enabled](https://github.com/keras-team/keras-cv/blob/f3e32b5d74d69419c5ec17b04f9ca96807474ed1/keras_cv/models/resnet_v1.py#L113), which causes redundant computation and is not aligned with other implementations.

I think this is rather a quick fix - add `use_bias=False` parameter to the Conv2d's in the `Block` function. If you approve I could submit a PR.

**Comment by sebastian-sz** [comment]
Gently pinging @LukeWood for opinion / green light here.

**Comment by LukeWood** [comment]
Interesting, could you send a PR so we can verify this?  I think you're correct, but I would like to see the updated code.

**Comment by sebastian-sz** [comment]
@LukeWood yes, submitted the PR here https://github.com/keras-team/keras-cv/pull/697
